nixos/modules/programs/direnv.nix

```
@@ -153,9 +153,7 @@ in
       ];
 
       etc = {
-        "direnv/direnv.toml".source = lib.mkIf (cfg.settings != { }) (
-          format.generate "direnv.toml" cfg.settings
-        );
+        "direnv/direnv.toml".source = format.generate "direnv.toml" cfg.settings;
```

Oops, missed that in review. But, you can also just move it this way
```suggestion
        "direnv/direnv.toml" = lib.mkIf (cfg.settings != { }) { 
          source = format.generate "direnv.toml" cfg.settings;
        };
```

---

Np, might need to apply a format too. I just did this in browser manually